op/model.py

```diff
@@ -72,6 +71,36 @@ def __init__(self, name, backend, cache):
 
         self._is_our_app = self.name == self._backend.app_name
 
+        self._status = None
+
+    @property
+    def status(self):
+        if not self._is_our_app:
+            return Unknown()
+
+        # TODO: do a leadership check (app status is not available to units for reading/writing).
```

Comment:
The leadership PR was merged, so this should be addressed.

Reply:
Done.